JapaneseFilter/sections/css_extended.txt

```diff
@@ -98,6 +98,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 !
 !
+odaibako.net#?#div[class^="sc-"] > div[class^="sc-"] > div[class^="sc-"] > div[class^="sc-"] > div[id^="div-gpt-"]:upward(4)
```

Comment:
It can be rewritten like
```
odaibako.net#?#.flex-grow > main > div[class^="sc-"]:has(> div[class^="sc-"] > div[class^="sc-"] div[id^="div-gpt-ad-"])
```